dcpquery/etl/__init__.py

```diff
@@ -110,7 +110,44 @@ def load_bundle(self, bundle, extractor=None, transformer=None):
         config.db_session.commit()
 
 
-def create_view_tables(extractor):
+def update_process_join_table():
+    config.db_session.execute(
+        """
+        Insert into process_join_table (child_process_uuid, parent_process_uuid)
+select * from (
```

Comment:
Please fix the indentation here to be consistent with the other lines.

Reply:
👍